In [46]:
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


In [47]:
import pandas as pd
import numpy as np

In [48]:
df1 = pd.read_csv('/content/drive/MyDrive/CODING/project2/combine_data_since_2000_PROCESSED_2018-04-26.csv')
df1.head()

Player  Pos  Ht   Wt  ...    AV                Team  Round   Pick
0      John Abraham  OLB  76  252  ...  26.0       New York Jets    1.0   13.0
1   Shaun Alexander   RB  72  218  ...  26.0    Seattle Seahawks    1.0   19.0
2    Darnell Alford   OT  76  334  ...   0.0  Kansas City Chiefs    6.0  188.0
3      Kyle Allamon   TE  74  253  ...   0.0                 NaN    NaN    NaN
4  Rashard Anderson   CB  74  206  ...   6.0   Carolina Panthers    1.0   23.0

[5 rows x 16 columns]

In [49]:
df1.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 6218 entries, 0 to 6217
Data columns (total 16 columns):
 #   Column     Non-Null Count  Dtype  
---  ------     --------------  -----  
 0   Player     6218 non-null   object 
 1   Pos        6218 non-null   object 
 2   Ht         6218 non-null   int64  
 3   Wt         6218 non-null   int64  
 4   Forty      6046 non-null   float64
 5   Vertical   4796 non-null   float64
 6   BenchReps  4212 non-null   float64
 7   BroadJump  4754 non-null   float64
 8   Cone       3993 non-null   float64
 9   Shuttle    4063 non-null   float64
 10  Year       6218 non-null   int64  
 11  Pfr_ID     4895 non-null   object 
 12  AV         6218 non-null   float64
 13  Team       3738 non-null   object 
 14  Round      3738 non-null   float64
 15  Pick       3738 non-null   float64
dtypes: float64(9), int64(3), object(4)
memory usage: 777.4+ KB


1) Source of data
<br>
https://www.kaggle.com/savvastj/nfl-combine-data
<br><br>
2) Brief description of data
<br>
The data is nfl combine data of how athletes performed and where they got drafted
<br><br>
3) What is the target?
<br>
the target will be the round the player got drafted
<br><br>
4) Is this a classification or regression problem?
<br>
Im actually going to try both ways and see what predicts better
<br><br>
5) How many features?
<br>
11 features after i drop some of the things that are not needed like team and pfr_id
<br><br>
6) How many rows of data.
<br>
6218
<br><br>
7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?
<br>
just getting a model to perform well. Interesting cleaning process will have to really think about how to fill some values or drop.

<br>
I know says pick a data set thats missing only 10 percent but after i drop some of the columns it will be close to that

In [50]:
df2 = pd.read_csv('/content/drive/MyDrive/CODING/project2/yahoo_stock.csv')
df2.head()

Date         High          Low  ...        Close        Volume    Adj Close
0  2015-11-23  2095.610107  2081.389893  ...  2086.590088  3.587980e+09  2086.590088
1  2015-11-24  2094.120117  2070.290039  ...  2089.139893  3.884930e+09  2089.139893
2  2015-11-25  2093.000000  2086.300049  ...  2088.870117  2.852940e+09  2088.870117
3  2015-11-26  2093.000000  2086.300049  ...  2088.870117  2.852940e+09  2088.870117
4  2015-11-27  2093.290039  2084.129883  ...  2090.110107  1.466840e+09  2090.110107

[5 rows x 7 columns]

In [51]:
df2.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1825 entries, 0 to 1824
Data columns (total 7 columns):
 #   Column     Non-Null Count  Dtype  
---  ------     --------------  -----  
 0   Date       1825 non-null   object 
 1   High       1825 non-null   float64
 2   Low        1825 non-null   float64
 3   Open       1825 non-null   float64
 4   Close      1825 non-null   float64
 5   Volume     1825 non-null   float64
 6   Adj Close  1825 non-null   float64
dtypes: float64(6), object(1)
memory usage: 99.9+ KB


1) Source of data
<br>
https://www.kaggle.com/arashnic/time-series-forecasting-with-yahoo-stock-price
<br><br>
2) Brief description of data
<br>
Yahoo stocks price over time day to day!
<br><br>
3) What is the target?
<br>
The target will be the CLose
<br><br>
4) Is this a classification or regression problem?
<br>
regression
<br><br>
5) How many features?
<br>
6
<br><br>
6) How many rows of data.
<br>
1825
<br><br>
7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?
<br>
Just trying to find something that works in models alot of exploring to decide what to do.